web/core/components/issues/peek-overview/issue-detail.tsx

```diff
@@ -100,7 +100,7 @@ export const PeekOverviewIssueDetails: FC<IPeekOverviewIssueDetails> = observer(
         isSubmitting={isSubmitting}
         setIsSubmitting={(value) => setIsSubmitting(value)}
         issueOperations={issueOperations}
-        disabled={disabled}
+        disabled={true}
```

Comment:
instead of this, leave it as it is.

at the top... just after disabled value is recieved from props, set disabled=True and add a comment just above it that "title is disabled everywhere" or something. so that IF AT ALL, it has to be conditionally enabled(worst case), we can do it off.

Reply:
Nevermind. it will disable other fields too. didnt notice that